less/v2/apps.less

```diff
@@ -1086,6 +1086,62 @@ dialog.dnd5e2.application {
   }
 }
 
+/* ---------------------------------- */
+/*  Config Sheets                     */
+/* ---------------------------------- */
+
+.dnd5e2.config-sheet {
+  .window-content > section:is(.flexcol, .flexrow) {
+    gap: 12px;
+  }
+}
+
+/* ---------------------------------- */
+/*  Hit Dice Config Sheet             */
+/* ---------------------------------- */
+
+.dnd5e2.hit-dice.config-sheet {
+  .card { padding-block: 8px; }
+  .form-group {
+    .separator { flex: 0; }
+    .max {
+      font-size: var(--font-size-12);
+      text-align: center;
+    }
+    [data-action="roll"] {
+      margin-inline-start: 8px;
+      padding-inline: 1em;
+    }
+    &:not(:last-of-type) {
+      border-block-end: var(--dnd5e-border-dotted);
+      padding-block-end: 4px;
+    }
+  }
+  input:invalid { outline: 1px solid var(--dnd5e-color-failure); }
+}
+
+/* ---------------------------------- */
+/*  Hit Points Config Sheet           */
+/* ---------------------------------- */
+
+.dnd5e2.hit-points.config-sheet {
+  .total-hp {
+    font-size: var(--font-size-32);
+    text-align: center;
+  }
+  .plusSign, .minusSign {
```

Comment:
I used these classes because `Intl.NumberFormat` uses camel case in its `type` values, which I use to automatically populate these in a couple of places. I could attempt to convert camel case to kebab case in that method if we think that would be cleaner overall.

Reply:
No that's fine if it's from `NumberFormat`